service/dps/api/src/main/java/eu/europeana/cloud/service/dps/ValidationStaticsReportService.java

```diff
@@ -0,0 +1,22 @@
+package eu.europeana.cloud.service.dps;
+
+import eu.europeana.cloud.common.model.dps.StatisticsReport;
+
+
+/**
+ * Service to get statistics report for Task Executions when task was performing validation step.
+ */
+public interface ValidationStaticsReportService {
```

Comment:
Class name is spelled incorrectly - it should be ValidationStatisticsReportService